package.json

```diff
@@ -3,16 +3,22 @@
   "version": "1.0.0",
   "main": "index.js",
   "repository": "https://github.com/hariharanweb/ondc-adapter.git",
+  "type": "module",
   "scripts": {
-    "start": "node --experimental-specifier-resolution=node src/app.js",
+    "start": "node --experimental-modules src/app.js",
     "test": "vitest --reporter verbose --globals",
     "test:coverage": "vitest run --coverage",
     "lint": "eslint src --fix"
   },
+  "dependencies": {
+    "csv-parse": "^5.3.6"
+  },
   "devDependencies": {
-    "eslint": "^8.36.0",
+    "eslint": "^8.29.0",
     "eslint-config-airbnb-base": "^15.0.0",
     "eslint-plugin-import": "^2.27.5",
+    "fs": "^0.0.1-security",
```

Comment:
doesnt sount right; there is no fs package.

Reply:
updated